opensearch/src/test/java/org/opensearch/sql/opensearch/storage/script/filter/FilterQueryBuilderTest.java

```diff
@@ -1821,7 +1821,7 @@ void cast_to_timestamp_in_filter() {
         "{\n"
             + "  \"term\" : {\n"
             + "    \"timestamp_value\" : {\n"
-            + "      \"value\" : 1636390800000,\n"
+            + "      \"value\" : \"2021-11-08 17:00:00\",\n"
```

Comment:
Instead of replacing value here, can you add  separate test with different formats? We should support all formats mentioned in https://opensearch.org/docs/latest/field-types/supported-field-types/date/

Reply:
Can you also add IT to cover this case?

Reply:
what is the issue with epoch value? why we want cast to ISO 8601 format?

Reply:
My intention with this PR was to remove additional formatting to specifically epoch on literal value during query build. If we expect literal value to be the right formatted value, then this additional default formatting can cause issues. 
Having said that today these literal values today are not honoring all the right date/time formats. See the formatter we are using today to format timestamp string in ExprTimeStamp.value class
https://github.com/opensearch-project/sql/blob/8eae36f69da0a95acb9f453c261396699904113d/core/src/main/java/org/opensearch/sql/data/model/ExprTimestampValue.java#L34

Its always `yyyy-MM-dd HH:mm:ss` for now
https://github.com/opensearch-project/sql/blob/8eae36f69da0a95acb9f453c261396699904113d/core/src/main/java/org/opensearch/sql/utils/DateTimeFormatters.java#L125

So in this case literal value is ISO 8601. But if we fix the format while parsing the date string in ExprTimeStampValue/ExprDateValue before using it in query builders, we do not have to format to epoch or any other default values.